package.json

```
@@ -65,7 +65,7 @@
     "@opengovsg/angular-legacy-sortablejs-maintained": "^1.0.0",
     "@opengovsg/angular-recaptcha-fallback": "^5.0.0",
     "@opengovsg/formsg-sdk": "0.8.3",
-    "@opengovsg/myinfo-gov-client": "^2.1.3",
+    "@opengovsg/myinfo-gov-client": "^3.0.0",
```

Let's lock the client version first in case we make backward-incompatiblie alterations to the types
```suggestion
    "@opengovsg/myinfo-gov-client": "=3.0.0",
```